.travis/environments/travis/public.yml

```diff
@@ -93,12 +93,16 @@ localsettings:
         class: 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow'
         instance: 'corehq.apps.userreports.pillow.get_kafka_ucr_static_pillow'
         params:
-          ucr_division: '09'
+          topics: ['case-sql']
+          include_ucrs:
+            - 'ucr2'
       - name: 'kafka-ucr-main-custom-2'
         class: 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow'
         instance: 'corehq.apps.userreports.pillow.get_kafka_ucr_static_pillow'
         params:
-          ucr_division: 'af'
```

Comment:
is this no longer used?

Reply:
This was only used as a stop gap when we got really behind on UCRs. it's still supported in the pillow, but we could probably remove it. It's not super useful anymore since we have a way to include/exclude specific UCR (for ICDS) and partitioning so we can split up pillows that way